src/Hypsography/Hypsography.jl

```diff
@@ -18,6 +18,24 @@ struct LinearAdaption{F <: Union{Fields.Field, Nothing}} <: HypsographyAdaption
     surface::F
 end
 
+"""
+    SLEVEAdaption(surface::Field, ηₕ::FT, s::FT)
+
+Locate vertical levels using an exponential function between the surface field and the top
+of the domain, using the method of [Schar2002](@cite). This method is modified
+such no warping is applied above some user defined parameter 0 ≤ ηₕ < 1.0, where the lower and upper 
+bounds represent the domain bottom and top respectively. `s` governs the decay rate. 
+If the decay-scale is poorly specified (i.e., `s * zₜ` is lower than the maximum 
+surface elevation), a warning is thrown and `s` is adjusted such that it `szₜ > maximum(z_surface)`. 
+"""
+struct SLEVEAdaption{F <: Union{Fields.Field, Nothing}, FT <: Real} <:
```

Comment:
Does this work with `F <: Nothing`? What is it intended to do in that case?

Reply:
This should throw an error with `<: Nothing `. A `Field` type is the supported input (this is not expected to be backwards compatible with the deprecation in `ExtrudedFiniteDifferenceSpaces.`). I have modified this such that `F <: Fields.Field`